### DISCUSSION TITLE
Allow .patch and .diff extensions for file uploads.

### DISCUSSION THREAD

**Discussion by abdullah2993** [community/community#6713] [Projects and Issues]
Right now we have to rename the patch file generated by `git format-patch` to `.patch.txt` to be able to upload it in an issue or pr review. Would be nice if we can directly upload the file without appending `.txt` to it.

![image](https://user-images.githubusercontent.com/5978905/138721810-fffe61e3-386b-42a6-abd5-fbec62f829cf.png)

**Comment by toastal** [comment]
For a platform for Git, it's embarrassing that proprietary software formats from PowerPoint and Excel are supported, but the VCS-relevant `*.patch` files are not.

**Comment by lmamane** [comment]
"PATCH" is now listed as one of the supported file types in the error message... but still get that error message when uploading a patch file (!!!)

EDIT: listed as "known issue" when using "Linux" on https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files
